draft-packages/tooltip/KaizenDraft/Tooltip/Tooltip.tsx

```diff
@@ -145,6 +156,18 @@ const TooltipContent = ({
   ) : null
 }
 
+const renderChildren = (
+  content: ReactNode,
+  tooltipId: string,
+  hasActiveTooltip: boolean
+): ReactNode => {
+  if (!hasSemanticRole(content)) return content
+
+  return cloneElement(content as ReactElement | React.ReactPortal, {
```

Comment:
I think we should be able to get to a state where we don't need to type-cast. Let's pair tomorrow? :)